packages/mobile/src/screens/app-drawer-screen/left-nav-drawer/LeftNavDrawer.tsx

```diff
@@ -101,21 +166,48 @@ const WrappedLeftNavDrawer = () => {
           onPress={() => {
             track(make({ eventName: Name.CHAT_ENTRY_POINT, source: 'navmenu' }))
           }}
-        >
-          {hasUnreadMessages ? (
-            <View style={styles.notificationBubble} />
-          ) : null}
-        </LeftNavLink>
+          showNotificationBubble={hasUnreadMessages}
+        />
       ) : null}
+      <LeftNavLink
+        icon={IconDonate}
+        label={messages.payAndEarn}
+        to='PayAndEarnScreen'
+        params={null}
+      >
+        <View style={styles.tokens}>
+          <LogoUSDC height={spacing(5)} width={spacing(5)} />
+          {isUsdcBalanceLoading ? (
+            <Skeleton style={styles.tokenAmount} height={18} width={24} />
```

Comment:
`spacing` here?

Reply:
i dont understand

Reply:
sorry, i meant `height=spacing(6)` instead of `height={18}`

Reply:
ah. cool